syntax = "proto3";

message response_CR {
  string Timestamp = 1;
  int32 Result = 2;
}

